core/src/main/scala/quasar/plugin/postgres/PostgresDestinationModule.scala

```diff
@@ -16,5 +16,132 @@
 
 package quasar.plugin.postgres
 
-object PostgresDestinationModule {
+import slamdata.Predef._
+
+import argonaut._, Argonaut._
+
+import cats.data.EitherT
+import cats.effect._
+import cats.implicits._
+
+import doobie._
+import doobie.hikari.HikariTransactor
+import doobie.implicits._
+
+import eu.timepit.refined.auto._
+
+import java.util.concurrent.Executors
+
+import quasar.api.destination.{DestinationError => DE, _}
+import quasar.concurrent.{BlockingContext, NamedDaemonThreadFactory}
+import quasar.connector.{DestinationModule, MonadResourceErr}
+
+import scala.concurrent.ExecutionContext
+import scala.util.control.NonFatal
+
+import scalaz.syntax.tag._
+
+object PostgresDestinationModule extends DestinationModule {
+
+  type InitErr = DE.InitializationError[Json]
+
+  // The duration, in seconds, to await validation of the initial connection.
+  val ValidationTimeoutSeconds: Int = 10
```

Comment:
```suggestion
  val ValidationTimeout: FiniteDuration = 10.seconds
```

Seems more declarative.